helm-charts/bytebase/values.yaml

```diff
@@ -7,7 +7,7 @@ bytebase:
     port: 8080
     existingPgURLSecret: ""
     existingPgURLSecretKey: ""
-    pg: postgresql://user:secret@host:port/dbname
+    pg:
```

Comment:
We prefer users to use external pg under helm, does KubeBlocks need to use the embed PostgreSQL?

Reply:
> We prefer users to use external pg under helm, does KubeBlocks need to use the embed PostgreSQL?

kubeblocks supports using the external pg by `kbcli addon enable bytebase --set bytebase.option.pg=...`. But if the user don't set the bytebase.option.pg, bytebase will init failed due to the unavailable DSN in the original helm values.yaml.

- We have a simple usage in https://github.com/apecloud/kubeblocks/pull/4702

Reply:
> But if the user don't set the bytebase.option.pg, bytebase will init failed

It is our expect behavior. Without persistent storage set, this is likely to result in the loss of metadata, whereas for other data (e.g., backups, etc.), the loss is tolerable.

Reply:
> > But if the user don't set the bytebase.option.pg, bytebase will init failed
> 
> It is our expect behavior. Without persistent storage set, this is likely to result in the loss of metadata, whereas for other data (e.g., backups, etc.), the loss is tolerable.

I Get it!